inductiva/simulators/xbeach.py

```diff
@@ -12,19 +13,26 @@ def __init__(self):
         super().__init__()
         self.api_method_name = "sw.xbeach.run_simulation"
 
-    def run(
-        self,
-        input_dir: types.Path,
-        sim_config_filename: Optional[str] = "params.txt",
-        on: Optional[types.ComputationalResources] = None,
-        storage_dir: Optional[types.Path] = "",
-        extra_metadata: Optional[dict] = None,
-    ) -> tasks.Task:
+    @meta.deprecated_arg(n_cores="n_vcpus")
+    def run(self,
+            input_dir: types.Path,
+            n_vcpus: Optional[int] = None,
+            use_hwthread: bool = None,
```

Comment:
The default value of `use_hwthread` should be `True` everywhere. Let's assume users want to use all vcpus they have available.